liwords-ui/src/gameroom/analyzer.tsx

```diff
@@ -461,13 +493,25 @@ export const Analyzer = React.memo((props: AnalyzerProps) => {
         title="Analyzer"
         className="analyzer-card"
         extra={
-          <Button
-            shape="circle"
-            icon={<BulbOutlined />}
-            type="primary"
-            onClick={handleExaminer}
-            disabled={examinerLoading}
-          />
+          <>
+            <div className="analyzer-controls">
```

Comment:
Yeah, it should be applying those styles on tablet only. I'll fix after work tonight (and combine the code section, since the html is the same both ways. Only the css and its parent change.)